CONTRIBUTING.md

```diff
@@ -220,14 +256,37 @@ that, you want to use the web extension profile in the launch setup.
 The default build target will allow you to debug the extension by providing the
 right sourcemaps.
 
+## Releasing
+
+`coq-lsp` is released using `dune-release tag` + `dune-release`.
```

Comment:
Actually this is only for the server.

Reply:
Oh, ok, since the list mentioned the client I figured this was more general 👌